### DISCUSSION TITLE
请问window 64 应该下载哪个安装包呢

### DISCUSSION THREAD

**Discussion by somethin4nothing** [qier222/YesPlayMusic#1771] [Q&A] [answered]
没有找到windows的安装包

**Comment by hawtim** [comment, marked as answer]
> 没有找到windows的安装包

[YesPlayMusic-Setup-0.4.5.exe](https://github.com/qier222/YesPlayMusic/releases/download/v0.4.5/YesPlayMusic-Setup-0.4.5.exe)
